client/src/components/StatusPill/index.tsx

```diff
@@ -0,0 +1,15 @@
+interface StatusPillProps {
+  status: string;
```

Comment:
Would it make sense to only allow `on` and `off`?

```suggestion
  status: 'on' | 'off';
```

Reply:
[de46652](https://github.com/CDCgov/dibbs-ecr-refiner/pull/187/commits/de4665247a5de577b83aa0508b53527f07c270c9)